### PR TITLE
refactor(postcss-ordered-values): Consolidate utility functions.

### PR DESCRIPTION
**Pull request by ben-eb**

### CODE REVIEW

File: packages/postcss-ordered-values/src/lib/isMathFunctionNode.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { both, compose, either, includes, prop, toLower } from 'ramda';
+import isFunctionNode from './isFunctionNode';
+import includedIn from './includedIn';
+
+const _isMathFunctionNode = compose(
+  either(includedIn(['min', 'max', 'clamp']), includes('calc')),
```
Comment (evilebottnawi) [inline packages/postcss-ordered-values/src/lib/isMathFunctionNode.js]: min/max/clamp like a calc function (new spec)

File: packages/postcss-ordered-values/src/lib/isVariableFunctionNode.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { both } from 'ramda';
+import isFunctionNode from './isFunctionNode';
+import isNodeValueOneOf from './isNodeValueOneOf';
+
+const isVariableFunctionNode = both(
+  isFunctionNode,
+  isNodeValueOneOf(['var', 'env'])
```
Comment (evilebottnawi) [inline packages/postcss-ordered-values/src/lib/isVariableFunctionNode.js]: `contant` (legacy `env`)

File: packages/postcss-ordered-values/src/lib/isMathFunctionNode.js (hunk 1 above)
Comment (ben-eb) [reply packages/postcss-ordered-values/src/lib/isMathFunctionNode.js]: So we should handle `-webkit-clamp()` etc?

Comment (evilebottnawi) [reply packages/postcss-ordered-values/src/lib/isMathFunctionNode.js]: can't find docs about this, but i think will be great always drop prefix

Comment (ben-eb) [reply packages/postcss-ordered-values/src/lib/isMathFunctionNode.js]: OK, will update. 👍

**Pull request closed without merging**